### PR TITLE
feat(style guide): New doc about getting set up to build the site locally

### PR DESCRIPTION
**Pull request by paperclypse**
I hope this fills a bit of the gap in our set up for new hires.

### CODE REVIEW

**Comment by paperclypse** [comment]
@clarkmcadoo You mind taking a look at this one to make sure I got all the details right? 
Regarding getting set up to do local builds.

File: src/content/docs/style-guide/writing-docs/writer-workflow/set-up-local.mdx [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: Set up your local build environment
+topics:
+  - Tech writer style guide
+  - Processes and procedures
+---
+
+Running the site locally makes testing and previewing large changes much easier. There are several things you need to do get this working. Fortunately, you only need to do this once.
+
+## 0. Enable VS Code in your terminal
```
Comment (bradleycamacho) [inline src/content/docs/style-guide/writing-docs/writer-workflow/set-up-local.mdx:10]: TIL it's `VS Code` and not `VSCode` 🤦🏼

File: src/content/docs/style-guide/writing-docs/writer-workflow/set-up-local.mdx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+It's handy to be able to open files from your terminal directly in VS Code. You may only use this a couple of times, but it's much nicer than trying to edit files in the terminal itself.
+
+1. Open Visual Studio Code.
+2. Press **command + shift + p** to open the command palette, then type "install code", and then click **Shell command: Install 'code' command in PATH**.
+3. In your terminal, type `code <var>filename</var>` to open that file in VS Code.
+
+## 1. Set up GitHub Desktop
+
+GitHub Desktop is a graphical tool for managing your branches, commits, and PRs. It's probably the most straightforward way to work with git and GitHub.
```
Comment (bradleycamacho) [inline src/content/docs/style-guide/writing-docs/writer-workflow/set-up-local.mdx:20]:
```suggestion
GitHub Desktop is a graphical tool for managing your branches, commits, and PRs. It's probably the most straightforward way to work with Git and GitHub.
```

File: src/content/docs/style-guide/writing-docs/writer-workflow/set-up-local.mdx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+## 0. Enable VS Code in your terminal
+
+It's handy to be able to open files from your terminal directly in VS Code. You may only use this a couple of times, but it's much nicer than trying to edit files in the terminal itself.
+
+1. Open Visual Studio Code.
+2. Press **command + shift + p** to open the command palette, then type "install code", and then click **Shell command: Install 'code' command in PATH**.
+3. In your terminal, type `code <var>filename</var>` to open that file in VS Code.
+
+## 1. Set up GitHub Desktop
```
Comment (bradleycamacho) [inline src/content/docs/style-guide/writing-docs/writer-workflow/set-up-local.mdx:18]: I'd mention that this section also downloads the repo locally. Maybe it's just me, but this header makes it seem like I'm just getting the app set up.

File: src/content/docs/style-guide/writing-docs/writer-workflow/set-up-local.mdx [hunk 4]
```diff
@@ -0,0 +1,97 @@
+---
+title: Set up your local build environment
+topics:
+  - Tech writer style guide
+  - Processes and procedures
+---
+
+Running the site locally makes testing and previewing large changes much easier. There are several things you need to do get this working. Fortunately, you only need to do this once.
```
Comment (bradleycamacho) [inline src/content/docs/style-guide/writing-docs/writer-workflow/set-up-local.mdx:8]: Might be overkill, but a section that covers how to open/use a terminal, and what it even is, may be helpful.

File: src/content/docs/style-guide/writing-docs/writer-workflow/set-up-local.mdx [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+Yarn is another package manager used to compile all of the files in the docs-website repo to build the site locally. You use npm to install Yarn.
+
+1. In your terminal, type `npm i -g yarn`.
+
+## Build the site locally
+
+Once you've installed all of these things in your terminal, you can run the site locally on your machine.
+
+1. In your terminal, type `yarn` to make sure all your Gatsby dependencies are up to date.
```
Comment (bradleycamacho) [inline src/content/docs/style-guide/writing-docs/writer-workflow/set-up-local.mdx:96]: Maybe a step to ensure they're in the right directory

**Review by bradleycamacho — DISMISSED** [review]

**Comment by bradleycamacho** [comment]
This looks great, just a few suggestions. Thanks for writing this up @paperclypse !!

**Pull request merged**